test/loader/test_neighbor_loader.py

```diff
@@ -54,7 +54,7 @@ def test_homogeneous_neighbor_loader(directed):
         assert is_subset(batch.edge_index, data.edge_index, batch.x, batch.x)
 
         # Test for isolated nodes (there shouldn't exist any):
-        assert data.edge_index.view(-1).unique().numel() == data.num_nodes
+        assert batch.edge_index.view(-1).unique().numel() == batch.num_nodes
```

Comment:
I assumed this was a mistake, can revert if not (otherwise same check is done for each batch)

Reply:
Thank you :)

Reply:
Actually.. this test fails now 😓  personally I'm not sure if its a problem if this is not true. There could be isolated nodes in the subgraph?